examples/msbr/openmc_msbr_model.py

```diff
@@ -0,0 +1,1045 @@
+                                        4.953,
+                                        0.46,
+                                        0.66802,
+                                        4.28498,
+                                        4.53898,
+                                        5.62102,
+                                        2.2225,
+                                        'gr_round_4',
+                                        optimized)
+    l = zoneIA(gr_sq_neg,
+               gr_extra_regions,
+               inter_elem_channel,
+               gr_round_4,
+               moder,
+               fuel,
+               hast,
+               optimized)
+    (gr_sq_neg,
+     gr_extra_regions,
+     inter_elem_channel,
+     gr_round_4) = shared_elem_geometry('core',
+                                        4.953,
+                                        0.46,
+                                        0.66802,
+                                        4.28498,
+                                        4.53898,
+                                        5.62102,
+                                        2.2225,
+                                        'gr_round_4',
+                                        optimized)
+
+    z = zoneIIA(gr_sq_neg,
+                gr_extra_regions,
+                inter_elem_channel,
+                gr_round_4,
+                moder,
+                fuel,
+                optimized)
+    v = void_cell()
+    # tres, uno, dos, quatro
+    t, u, d, q = graphite_triangles(fuel, moder)
+
+    s1, s2, s3 = zone_i_boundary
+
+    main = openmc.RectLattice()
+    main.pitch = np.array([10.16, 10.16])
+    N = 45 / 2
+    main.lower_left = -1 * main.pitch * N
+    main.universes = [[v, v, v, v, v, v, v, v, v, v, v, v, v, v, v, v, v, d, z, z, z, z, z, z, z, z, z, u, v, v, v, v, v, v, v, v, v, v, v, v, v, v, v, v, v],
+                      [v, v, v, v, v, v, v, v, v, v, v, v, v, d, z, z, z, z, l, l, l, l, l, l, l, l, l, z, z, z, z, u, v, v, v, v, v, v, v, v, v, v, v, v, v],
+                      [v, v, v, v, v, v, v, v, v, v, v, d, z, z, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, z, z, u, v, v, v, v, v, v, v, v, v, v, v],
+                      [v, v, v, v, v, v, v, v, v, v, d, z, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, z, u, v, v, v, v, v, v, v, v, v, v],
+                      [v, v, v, v, v, v, v, v, v, d, z, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, z, u, v, v, v, v, v, v, v, v, v],
+                      [v, v, v, v, v, v, v, v, d, z, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, z, u, v, v, v, v, v, v, v, v],
+                      [v, v, v, v, v, v, v, d, z, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, z, u, v, v, v, v, v, v, v],
+                      [v, v, v, v, v, v, d, z, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, l, z, u, v, v, v, v, v, v],
```

Comment:
There *has* to be a better way than this...

Reply:
If you can think of one let me know.

Reply:
@munkm @smpark7 I'm not familiar with openMC, do you know of a better way to handle this?

Reply:
Does it follow a set pattern you could generate iteratively, maybe?

Reply:
Does this follow a pattern you could use to generate these arrays iteratively, maybe?

Reply:
What is the shape you're trying to make here?

Reply:
It's octagonal in nature but with some additional complexity.

![msbr_full_xy_openmc](https://user-images.githubusercontent.com/45364492/205745787-75af2e6c-5f6c-4c60-ad5a-8e8e028cd5ca.png)

Reply:
> Does this follow a pattern you could use to generate these arrays iteratively, maybe?

Now that I'm looking at it, I could, but this isn't a super high priority right now. I'll make an issue to address this.

Reply:
I think that's fine, then. I wanted to see the shape you were going for and maybe check for some typos. It's another one of those things where a small error could generate considerable confusion later on.